benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/scheduler/TestTaskScheduler.java

```diff
@@ -383,34 +439,21 @@ private synchronized void handleExperimentResult(String testID) {
     HandleExperimentResultTask task = new HandleExperimentResultTask(testID);
 
     // TODO - should go into a stateless queue (so that we can recover)
-    Future<HandleExperimentResultTask.Result> future = taskExecutorService.submit(task);
+    Future<Boolean> future = taskExecutorService.submit(task);
```

Comment:
Done in https://github.com/benchflow/benchflow/pull/394/commits/ab816423446799a9f6e388d52f85c15bef9a4c58